### ISSUE TITLE
Installer: Concurrent downloads?

### ISSUE THREAD

**Issue by enumag** [07th-mod/higurashi-patch-compiler#10]
@ItaloKnox Just an idea. The installer is downloading several rather large files. This could be a bit faster if they were downloaded concurrently. Is that possible?

(You can close this if you don't want to do it. It's just and idea and not too important.)

**Comment by ItaloKnox** [comment]
I can make it download several files at the same time but I kinda dislike it because it messes with the structure and makes it hard to follow. Though there are a couple improvements I could make here:

* Allow multiple connections (should have been there already but I forgot) << this would make the patch download way faster overall
* Put the two graphics patches in the same category and have them download at the same time << this will make the installation *look* shorter but not very efficient once we unlock all the connections

If that's good enough for you I can push them later tonight.

**Comment by enumag** [comment]
Sounds good. :-)

**Comment by ItaloKnox** [comment]
Good, I'll prepare the updates and give it a little test before pushing them.
